soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -1584,5 +1584,6 @@ void DebugConsole_Init(void) {
         { "reset|randomize", Ship::ArgumentType::TEXT },
     }});
 
+    CVarSave();
```

Comment:
not clear why this is being added, anyone have context as to why we have this or load here?